packages/editor/src/components/post-schedule/label.js

```diff
@@ -2,14 +2,14 @@
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { dateI18n, __experimentalGetSettings } from '@wordpress/date';
+import { dateI18n } from '@wordpress/date';
 import { withSelect } from '@wordpress/data';
 
 export function PostScheduleLabel( { date, isFloating } ) {
-	const settings = __experimentalGetSettings();
+	const formatWithTimezone = 'M j, Y g: i a T';
```

Comment:
We are defining the formats in:
https://github.com/WordPress/gutenberg//blob/64c48fdea724843009b48c373649c6382957eb6f/packages/date/src/index.js#L26 

What if we defined a new format datetimeAbbreviatedWithTimeZone in the same file? Doing this would keep all the formats in a central location, and avoid having a hardcoded format in this file.

Reply:
I would be fine with that. I did try that at first approach, but it wasn't working for me. I'm not sure if it was a npm packages issue I had with building or not, but I could not get the new format to show in the editor